File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
       unordered_map<int, int> Map;
       for(int num:nums){
        if(Map.find(num)!=Map.end()){
           return true;
        }
        Map[num]++;
       }
       return false; 
    }
};

File: Data Structures & Algorithms/sort-array-by-parity/submission-1.py
class Solution:
    def sortArrayByParity(self, nums: List[int]) -> List[int]:
        j = 0
        n=len(nums)
        for i in range(n):
            if nums[i]%2==0:
                temp = nums[j]
                nums[j] = nums[i]
                nums[i] = temp
                j+=1
        return nums